nh_eobs_mental_health/models/nh_eobs_ward_dashboard.py

```diff
@@ -86,6 +96,8 @@ def init(self, cr):
                 capacity=sql_statements.get_ward_dashboard_capacity_count(),
                 obs_stop=sql_statements.get_ward_dashboard_obs_stop_count(),
                 on_ward=sql_statements.get_ward_dashboard_on_ward_count(),
-                dashboard=sql_statements.get_ward_dashboard()
+                refused_obs_count=ref_count,
```

Comment:
I would just have the `sql_statements.get_ward_dashboard_refused_obs_count()` here to keep it consistent with the others, doesn't really add much having the variable.